### ISSUE TITLE
[Magento] Unable to find configuration to make Magento aware of domain

### ISSUE THREAD

**Issue by alymcmurray** [bitnami/vms#262]
### Describe your issue as much as you can

I have been following guides to get Magento up and running using Lightsail on AWS.
Here are the guides I am referencing:
https://aws.amazon.com/getting-started/hands-on/magento-on-aws/
https://lightsail.aws.amazon.com/ls/docs/en_us/articles/amazon-lightsail-quick-start-guide-magento#amazon-lightsail-magento-attach-static-ip
https://docs.bitnami.com/general/how-to/troubleshoot-magento-issues/

All three of these guides inform me that I need to make Magento aware of the static IP by using one of the following commands:

`sudo /opt/bitnami/configure_app_domain --domain IP_OR_DOMAIN`
`sudo /opt/bitnami/apps/magento/bnconfig --machine_hostname IP_OR_DOMAIN`

Neither of these commands work, I get a Command Not Found error. I will provide screenshots.

I've got the instance created in lightsail, I've got a static IP set. I've got the application password, which shouldn't be needed at this step. I have deleted and re-started from scratch (multiple times). Can you help me figure out what to do?

Below is the screenshot of hopefully relevant directory listings, plus the command not found error. 

![image](https://user-images.githubusercontent.com/35636146/185636688-9087eddb-2e5b-4136-bc56-421840537c9d.png)

Thank you in advanced for your assistance. This is making me feel seriously dull, like this should not be an issue for me, and I'm wondering what overly-simple step I'm missing.

**Comment by mdhont** [comment]
Can you check this guide:

https://docs.bitnami.com/aws/faq/configuration/configure-custom-domain/#update-application-configuration

The bnconfig tool is deprecated. On newer machines you need to run the configure_app_domain script to configure the app's domain:

    sudo /opt/bitnami/configure_app_domain --domain DOMAIN-NAME

**Comment by alymcmurray** [comment]
> Can you check this guide:
> 
> https://docs.bitnami.com/aws/faq/configuration/configure-custom-domain/#update-application-configuration
> 
> The bnconfig tool is deprecated. On newer machines you need to run the configure_app_domain script to configure the app's domain:
> 
> ```
> sudo /opt/bitnami/configure_app_domain --domain DOMAIN-NAME
> ```

If you see in my original post, I tried that first and went to the alternative/backup option afterwards.
See the screenshot attached - neither bnconfig nor configure_app_domain are located in their resepective folders after using the ls command. 

I created a new instance on a higher tier plan and it appeared - it seems the issue lies on the minimum tier plan, as if it's not able to create that file for some particular reason, as I tried at least 3 separate instances on lower tier plans and it did not populate the file; but twice on upper tier plans it did populate the file.
